[showdesktop] Fix sizing of the plasmoid in the panel
 see https://quickgit.kde.org/?p=kdeplasma-addons.git&a=commit&h=e164ae7c02f5579e02d26842bee6a4a450e01871
 FIXED-IN: 5.6.1
 REVIEW: 127499

--- a/applets/showdesktop/package/contents/ui/main.qml
+++ b/applets/showdesktop/package/contents/ui/main.qml
@@ -33,8 +33,8 @@
 
     property bool vertical: (plasmoid.formFactor == PlasmaCore.Types.Vertical)
 
-    Layout.minimumWidth: Layout.maximumWidth
-    Layout.minimumHeight: Layout.maximumHeight
+    Layout.minimumWidth: units.iconSizes.small
+    Layout.minimumHeight: Layout.minimumWidth
 
     Plasmoid.preferredRepresentation: Plasmoid.fullRepresentation
     Plasmoid.onActivated: showdesktop.showDesktop();
